Topics/Introduction_To_Kubernetes.md

```diff
@@ -0,0 +1,125 @@
+# Kubernetes
+
+## Table of Contents
+1. [Introduction](#Introduction)
+    - [What is Kubernetes(k8s)?](#What-is-Kubernetes(k8s)?)
+    - [Kubernetes vs Docker](#k8s-vs-docker)
+    - [Why k8s?](#why-k8s)
+2. [CI/CD](#CI/CD)
+3. [Examples](#Examples)
+
+## Introduction
+### What is Kubernetes(k8s)?
+
+Kubernetes known as k8s is a cluster manage tool and distrubuted supporting platform to use automate, deploying, and scaling container.
+
```

Comment:
Maybe briefly explain what containers are to someone who is unfamiliar with the technology

Reply:
is explained in the docker section